Remove unnecessary C file compilation from grpcio_observability.
Replaced by dynamic linking.
--- a/src/python/grpcio_observability/setup.py
+++ b/src/python/grpcio_observability/setup.py
@@ -219,7 +219,7 @@
     EXTRA_LINK_ARGS.append("-static-libstdc++")
 
 CC_FILES = [
-    os.path.normpath(cc_file) for cc_file in observability_lib_deps.CC_FILES
+    #os.path.normpath(cc_file) for cc_file in observability_lib_deps.CC_FILES
 ]
 CC_INCLUDES = [
     os.path.normpath(include_dir)
